src/SignalR/server/Core/src/Internal/DefaultHubDispatcherLog.cs

```diff
@@ -105,4 +105,7 @@ public static void ClosingStreamWithBindingError(ILogger logger, CompletionMessa
 
     [LoggerMessage(24, LogLevel.Debug, "CompletionMessage for invocation ID '{InvocationId}' received unexpectedly.", EventName = "UnexpectedCompletion")]
     public static partial void UnexpectedCompletion(ILogger logger, string invocationId);
+
+    [LoggerMessage(25, LogLevel.Error, "InvocationId {InvocationId}: Failed during streaming '{HubMethod}'.", EventName = "FailedStreaming")]
```

Comment:
nit:
```suggestion
    [LoggerMessage(25, LogLevel.Error, "InvocationId '{InvocationId}': Failed during streaming '{HubMethod}'.", EventName = "FailedStreaming")]
```

Reply:
Some have quotes:

https://github.com/dotnet/aspnetcore/blob/c9bd468c2075678d3b764ff9159c25db17cd7049/src/SignalR/server/Core/src/Internal/DefaultHubDispatcherLog.cs#L103

https://github.com/dotnet/aspnetcore/blob/c9bd468c2075678d3b764ff9159c25db17cd7049/src/SignalR/server/Core/src/Internal/DefaultHubDispatcherLog.cs#L106

And some have no quotes:

https://github.com/dotnet/aspnetcore/blob/c9bd468c2075678d3b764ff9159c25db17cd7049/src/SignalR/server/Core/src/Internal/DefaultHubDispatcherLog.cs#L35

https://github.com/dotnet/aspnetcore/blob/c9bd468c2075678d3b764ff9159c25db17cd7049/src/SignalR/server/Core/src/Internal/DefaultHubDispatcherLog.cs#L47

https://github.com/dotnet/aspnetcore/blob/c9bd468c2075678d3b764ff9159c25db17cd7049/src/SignalR/server/Core/src/Internal/DefaultHubDispatcherLog.cs#L56

I copied it from the ones without quotes :)
Let's wait if a new `LoggerMessage` is even needed based on the ongoing discussion above.

Reply:
Bike-shed time.

`"Invocation ID {InvocationId}: Failed while sending stream items from hub method {HubMethod}."`